source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -2466,6 +2466,17 @@ configuration settings.
 
       This configuration setting corresponds with the :envvar:`MINIO_IDENTITY_OPENID_CLIENT_SECRET` environment variable.
       
+   .. mc-conf:: role_policy
+      :delimiter: " "
+
+      *Optional*
+
+      .. include:: /includes/common-minio-external-auth.rst
+         :start-after: start-minio-openid-role-policy
+         :end-before: end-minio-openid-role-policy
+
```

Comment:
This setting is mutually exclusive with the claim_name setting - need to specify that for both config and envvar.